src/tests/ftest/soak/utils.py

```diff
@@ -181,10 +181,8 @@ def get_remote_dir(self, source_dir, dest_dir, host_list, shared_dir=None,
                     "<<FAILED: Soak remote logfiles not copied from clients>>: {}".format(
                         host)) from error
             command = "/usr/bin/cp -R -p {0}/ \'{1}\'".format(shared_dir_tmp, dest_dir)
-            try:
-                run_command(command, timeout=30)
-            except DaosTestError as error:
-                raise SoakTestError("<<FAILED: job logs failed to copy>>") from error
+            run_command(command, timeout=30, raise_exception=False)
```

Comment:
We should reinstate the `SoakTestError` raising for non-tearDown calling and instead use a `try .. except ..` in the `pre_tear_down()` method to capture any errors and perform the appropriate action (log a message, or add the error to the list of errors to be reported at the end of tearDown, etc.)